browser_use/browser/session.py

```diff
@@ -692,6 +695,7 @@ async def new_tab(self, url: str | None = None) -> Page:
 
 	@require_initialization
 	async def switch_tab(self, tab_index: int) -> Page:
+		assert self.browser_context is not None, 'BrowserContext object is not set'
```

Comment:
```suggestion
```
the `@require_initialization` decorator already checks that browser_context is set